legal-api/src/legal_api/services/authz.py

```diff
@@ -238,20 +226,20 @@ def has_roles(jwt: JwtManager, roles: List[str]) -> bool:
         },
         Business.State.HISTORICAL: {
             'courtOrder': {
-                'legalTypes': ['SP', 'GP', 'CP', 'BC', 'BEN', 'CC', 'ULC', 'LLC'],
+                'legalTypes': ['SP', 'GP', 'CP', 'BC', 'BEN', 'CC', 'ULC'],
             },
             'putBackOn': {
                 'legalTypes': ['SP', 'GP', 'BEN', 'CP', 'BC', 'CC', 'ULC'],
                 'blockerChecks': {
                     # FUTURE add outstanding filings to invalidStateFilings when implemented
```

Comment:
can we remove this comment?  `# FUTURE add outstanding filings to invalidStateFilings when implemented`